### PR TITLE
Clean up repo and update README in preparation for v1.0.0 release

### PR DESCRIPTION
**Pull request by toph-allen**
Does what it says on the tin.

### CODE REVIEW

File: fastapitableau/rstudio_connect.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def warning_message() -> Optional[str]:  # noqa: C901
     message_list: List[str] = []
 
     connect_server = environ.get("CONNECT_SERVER")
-    # TODO DEBUG: Print CONNECT_SERVER variable
+    # Debug message: Print CONNECT_SERVER variable
```
Comment (sagerb) [inline fastapitableau/rstudio_connect.py:32]: What is with these?

File: fastapitableau/rstudio_connect.py [hunk 2]
```diff
@@ -66,14 +65,13 @@ def warning_message() -> Optional[str]:  # noqa: C901
...
-        # TODO
-        # DEBUG: Add a message if API key not defined
+        # Debug message
 
     if len(message_list) != 0:
         messages = "\n\n---\n\n".join(message_list)
         return messages
 
-    # call API to see if thing is enabled.
+    # Call Connect server's API to get settings
```
Comment (sagerb) [inline fastapitableau/rstudio_connect.py:74]:
```suggestion
    # Call RStudio Connect API to get server settings
```

**Review by sagerb — APPROVED** [review]
minor nits. LGTM

File: fastapitableau/rstudio_connect.py (hunk 1 above)
Comment (toph-allen) [reply fastapitableau/rstudio_connect.py:32]: I wanted to remove the "TODO" comments, but leave in a marker for myself about where to post log messages from when I implement debug logging.

Comment (toph-allen) [reply fastapitableau/rstudio_connect.py]: Removed

**Pull request merged**